src/MeetingsSDKAdapter.js

```diff
@@ -806,21 +802,27 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       }),
     );
 
-    const meetingWithMediaStoppedSharingLocalEvent$ = fromEvent(sdkMeeting, EVENT_LOCAL_SHARE_STOP)
-      .pipe(
-        // TODO (SPARK-152004): This is a workaround for a bug of stopping share when onended event is triggered.
-        flatMap(() => defer(() => this.meetingControls[SHARE_CONTROL].action(ID))),
-        // eslint-disable-next-line no-console
-        tap(() => console.log('EVENT_LOCAL_SHARE_STOP is triggered', this)),
-        map(() => inactiveShare),
-      );
+    const meetingWithMediaStoppedSharingLocalEvent$ = fromEvent(
+      sdkMeeting,
+      EVENT_LOCAL_SHARE_STOPPED,
+    ).pipe(
+      // eslint-disable-next-line no-console
+      tap(() => console.log('EVENT_LOCAL_SHARE_STOPPED was triggered', this)),
```

Comment:
This should probably be informational
```suggestion
      tap(() => console.info('EVENT_LOCAL_SHARE_STOPPED was triggered', this)),
```